docs/deploy_api.md

```diff
@@ -0,0 +1,90 @@
+## API Deployment
+
+This document provides a step-by-step guide to deploy the `space2stats` API using AWS CDK.
+
+### 1. Dependencies Setup
+
+Before proceeding with the deployment, ensure you have all necessary dependencies installed. Specifically, you’ll need:
+
+- AWS CDK: Follow the installation guide [here](https://docs.aws.amazon.com/cdk/v2/guide/getting_started.html).
+- AWS CLI: Ensure that the AWS CLI is installed and configured.
+
```

Comment:
here we can tell to specifically how to install the dependency 


### Deployment

Install python dependencies with

```
# Create a virtual environment
python -m pip install --upgrade virtualenv
virtualenv .venv
source .venv/bin/activate

python -m pip install -r requirements-cdk.txt
```

And node dependencies with

```
npm install
```

Verify that the `cdk` CLI is available. Since `aws-cdk` is installed as a local dependency, you can use the `npx` node package runner tool, that comes with `npm`.

```
npx cdk --version
```

First, synthesize the app

```
npx cdk synth --all
```

Then, deploy

```
npx cdk deploy --all
```

Reply:
Added!